SourceX/storm/storm.cpp

```diff
@@ -40,13 +40,6 @@ radon::File &getIni()
 	return ini;
 }
 
-BOOL SFileDdaSetVolume(HANDLE hFile, signed int bigvolume, signed int volume)

```

Comment:
Interesting how only one parameter was actually being used...

Reply:
a combination of us only having to implement what diablo uses and having it all build as one big project.